### PR TITLE
perlcritic fixes

### PR DESCRIPTION
**Pull request by dkj**

### CODE REVIEW

File: Build.PL [hunk 1]
```diff
@@ -289,6 +289,7 @@ my $requires = {
                 'Digest::SHA1'                    => '0.0',
                 'English'                         => '1.02',
                 'MIME::Base64'                    => 0,
+                'perl'                            => '5.14',
```
Comment (mgcam) [inline Build.PL]: perl Build.PL 
Checking prerequisites...
  requires:
    !  perl (5.16.3) is installed, but we need version >= 5.14

'perl' => '5.14.0' is OK

Comment (dkj) [inline Build.PL]: Thanks.



-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.

File: cgi-bin/npg_qc [hunk 2]
```diff
@@ -1,18 +1,20 @@
 #!/usr/local/bin/perl -T
+use strict;
+use warnings;
```
Comment (mgcam) [inline cgi-bin/npg_qc]: There might be a reason why these two imports are not at the beginning of the file. This probably was important for the version of perl on the webteam's machines. As far as I remember, something to do with the warning module version.

File: cgi-bin/npg_qc [hunk 3]
```diff
@@ -1,18 +1,20 @@
...
+      my ($dtl) = $l =~ /([\w\/\.\-]+)/smxia;
+      #'a' modifiler for ascii only interpretation of \w
       if ($dtl) {push @npg_libs, $dtl;}
     }
   }
 }
 
-use strict;
 use lib @npg_libs;
 use warnings;
```
Comment (mgcam) [inline cgi-bin/npg_qc]: If we end up having import of strict and warnings modules at the top of the page, this line shoudl be deleted.

**Comment by mgcam** [comment]
Looked at old commit messages, did not find anything relevant. This code was introduced 4.5 years ago. I think we should merge the change.

**Comment by dkj** [comment]
Rebasing from 89b11a7cc68e2c847d4832c05a9d1ec7eced0a67

**Pull request merged**